microsoft/PowerToys
# Remove CreateScheduledTask CA and handle task creation from runner code

## Summary of the Pull Request
From what I see, CA is not needed as the runner can handle everything. Am I missing something? I've tested different scenarios and everything still works as expected.

Also some minor cleanup and refactoring.


## PR Checklist



## Detailed Description of the Pull Request / Additional comments


## Validation Steps Performed
- Install this version of PT on clean machine and check if scheduled task is created and working
- Try update scenario and see if scheduled task works correctly
